AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/test/Mutations/TestMutationHappyPath.dfy

```diff
@@ -84,6 +84,87 @@ module {:options "/functionSyntax:4" } TestMutationHappyPath {
     var _ := CleanupItems.DeleteBranchKey(Identifier:=branchKeyIdentifier, ddbClient:=ddbClient);
   }
 
+  method MutationRecoveryTest(
```

Comment:
This test/helper asserts:
- that Mutation Index can be deleted and recreated
- that the token can be retrieved via InitMutation 
We should put a comment detailing this.